### ISSUE TITLE
Azkend 2: The World Beneath [PCSE00861]

### ISSUE THREAD

**Issue by elPrimoDorado** [Vita3K/compatibility#1317]
# App summary
- App name: Azkend 2: The World Beneath
- App serial: PCSE00861
- App version: 1.00

# Vita3K summary
- Version: v0.1.6
- Build number: 3038
- Commit hash: https://github.com/vita3k/vita3k/commit/08dec7c
- CPU backend: Dynarmic
- GPU backend: Vulkan                           

# Test environment summary
- Tested by: elPrimoDorado
- OS: Windows 10
- CPU: AMD Ryzen 3 2200G
- GPU: NVIDIA 1050TI
- RAM: 32 GB

# Issues

Playable with no issues

# Screenshots

![Vita3K v0 1 6 3038-08dec7ca _ Azkend 2 (PCSE00861) _ Vulkan _ 60 FPS (17 ms) _ 960x544  02_03_2023 16_58_11](https://user-images.githubusercontent.com/39572347/222570540-5ad92908-d13f-4503-92f7-8a95ef450882.png)
![Vita3K v0 1 6 3038-08dec7ca _ Azkend 2 (PCSE00861) _ Vulkan _ 60 FPS (17 ms) _ 960x544  02_03_2023 16_56_29](https://user-images.githubusercontent.com/39572347/222570588-962eebf2-3a87-4593-954e-2aaa7f18fae7.png)

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/10876223/vita3k.log)

# Recommended labels

- Playable

**Comment by Slade1972** [comment]
This version should be checked to ensure it doesn't soft lock on completion of the Binocular mini game section (the first 4 tutorial parts). The European version will not progress past completing the Binocular tutorial, although it appears fully playable up until that point.

**Comment by github-user1112** [comment]
Should just be called Azkend 2 like the other regions and like what it says in header of game window

**Comment by thomyyg07** [comment]
i got soft lock, i think going to move the ingame - (now)
not Playable, wrong!
<img width="1202" height="719" alt="Image" src="https://github.com/user-attachments/assets/6688902c-afe1-4dda-8296-fe44137621b5" />

log: 

[vita3k.log](https://github.com/user-attachments/files/29665824/vita3k.log)

**Comment by github-user1112** [comment]
yeah same issue as other region https://github.com/Vita3K/compatibility/issues/2747
